lottie/src/main/java/com/airbnb/lottie/utils/Logger.java

```diff
@@ -0,0 +1,26 @@
+package com.airbnb.lottie.utils;
+
+/**
+ * Singleton object for logging. If you want to provide a custom logger implementation,
+ * implements LottieLogger interface in a custom class and replace Logger.instance
+ */
+public class Logger {
+  
+  public static LottieLogger instance = new LogcatLogger();
```

Comment:
If this is static, can you rename this `INSTANCE` and create a `static setInstance` function so it's consistent with other Lottie public APIs?

Reply:
Will fix